src/Microsoft.ML.AutoML/Sweepers/ISweeper.cs

```diff
@@ -230,7 +230,7 @@ IComparable IRunResult.MetricValue
 
     /// <summary>
     /// The metric class, used by smart sweeping algorithms.
-    /// Ideally we would like to move towards the new IDataView/ISchematized, this is
```

Comment:
This is more a "TODO" comment than documentation, but since ISchema doesn't exist anymore, I guess it doesn't really matter if I simply remove this mention of it.

Reply:
Yes. Also, you don't need the "new" anymore :-).

---
In reply to: [368201339](https://github.com/dotnet/machinelearning/pull/4673#discussion_r368201339) [](ancestors = 368201339)